### PR TITLE
Link to Showdown dex in /data

### PR DESCRIPTION
**Pull request by xfix**

### CODE REVIEW

File: style/utilichart.css [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	padding-top: 1px;
 }
 
-.utilichart li a
+.utilichart li > a, .utilitychart li > div
```
Comment (Slayer95) [inline style/utilichart.css]: Hmm, we don't have a ``.utilitychart``

Comment (xfix) [inline style/utilichart.css]: I fail... wait... how does it work then.

**Comment by monsanto** [comment]
> Link to XY Smogon analysis in /data to Link to Showdown dex in /data

wow, thanks for the show of support everyone

**Comment by Slayer95** [comment]
@monsanto, we are still planning to link to Smogdex from Teambuilder.

In other news, wasn't the plan to link moves and abilities to PSdex? :|

**Pull request merged**